Jenkinsfile

```diff
@@ -0,0 +1,199 @@
+pipeline_name="vivarium_gates_nutrition_optimization"
+conda_env_name="${pipeline_name}-${BUILD_NUMBER}"
+conda_env_path="/tmp/${conda_env_name}"
+// defaults for conda and pip are a local directory /svc-simsci for improved speed.
+// In the past, we used /ihme/code/* on the NFS (which is slower)
+shared_path="/svc-simsci"
+
+pipeline {
+  // This agent runs as svc-simsci on node simsci-slurm-sbuild-p01.
+  // It has access to standard IHME filesystems and singularity
+  agent { label "svc-simsci" }
+
+  options {
+    // Keep 100 old builds.
+    buildDiscarder logRotator(numToKeepStr: "100")
+
+    // Wait 60 seconds before starting the build.
+    // If another commit enters the build queue in this time, the first build will be discarded.
+    quietPeriod(60)
```

Comment:
Once we're running this on push/pull requests, we probably want to remove the quiet period?

Reply:
I'm not real sure tbh. I noticed that everyone around here always has a quietPeriod and I never reallly understood why.